### ISSUE TITLE
Some funky scrolling issues on iOS Safari

### ISSUE THREAD

**Issue by grese** [elwayman02/jordan-hawker#317]
When you view the site in Safari on iOS there are some scrolling issues.
- Content overlaps the top navigation bar.  This makes navigating the site impossible unless you scroll all the way to the top.
- Sometimes content gets clipped off at the bottom (this is kind of intermittent)

### Overlapping Navigation bar
<img width="406" alt="screen shot 2018-02-25 at 10 45 22 am" src="https://user-images.githubusercontent.com/4702431/36645161-9c530ee4-1a19-11e8-84e7-7fb87ebbbb78.png">

### Clipped content
<img width="405" alt="screen shot 2018-02-25 at 10 45 56 am" src="https://user-images.githubusercontent.com/4702431/36645168-ad04400a-1a19-11e8-9581-d138b189e1a9.png">

**Comment by grese** [comment]
I didn’t “view source” on your site yet, but if you don’t have this meta tag yet, adding it might help:

<meta name="viewport" content="width=device-width, initial-scale=1.0">

**Comment by grese** [comment]
I didn’t check the “view source” yet, but if you haven’t set the “viewport” meta tag yer, that might help.
`<meta name="viewport" content="width=device-width, initial-scale=1.0">`

**Comment by elwayman02** [comment]
That's an interesting one...might just need to play around with the z-index to guarantee they layer properly in every browser. Man, it doesn't matter what app I'm building, 100% of the time Safari is the buggiest one to deal with. 😭

**Comment by grese** [comment]
Agreed.  Mobile Safari is even trickier than desktop (and more difficult to automate without Saucelabs).  I fear that Safari is headed down the path of Internet Exploder.

Z-index will probably fix the overlapping navigation issue.

As for the content getting clipped, I’m not sure what’s going on there.  It almost seemed as if there was an “overflow:hidden” somewhere, but the problem is intermittent.

How are you embedding all of that 3rd party content?  Are they in iframes, or are you dynamically generating elements?

Kudos on using sha-256 to allow scripts via CSP.  That’s a pretty good security mechanism that should prevent a lot of XSS.

**Comment by elwayman02** [comment]
A few pieces of content are in iframes, but the majority is loaded from the public APIs via Ember-Data

**Comment by elwayman02** [comment]
The z-index issue should be resolved now, not sure about the clipping.
